CONTRIBUTING.md

```diff
@@ -17,38 +13,28 @@ You can also suggest anything else that you think improves the community for us
 
-We use [GitHub issues](https://github.com/kedro-org/kedro/issues) to keep track of known bugs. We keep a close eye on them and update them when we have an internal fix in progress.
 
-Before you report a new issue, do your best to ensure your problem hasn't already been reported. If it has, just leave a comment on the existing issue, rather than create a new one. If you have already checked the existing [GitHub issues](https://github.com/kedro-org/kedro/issues) and are still convinced that you have found odd or erroneous behaviour then please file a new one.
+## Join our Technical Steering Committee
 
-## Propose a new feature
-If you have new ideas for Kedro functionality then please open a [GitHub issue](https://github.com/kedro-org/kedro/issues) and describe the feature you would like to see, why you need it, and how it should work.
+Kedro is an incubating project in [LF AI & Data](https://lfaidata.foundation/), a sub-organisation within the Linux Foundation that focuses on open innovation within the data and AI space. The project is governed by a group of maintainers known as the Technical Steering Committee (TSC). You can read more about the structure of our TSC in our [Technical Charter](TODO).
 
-
-## Review community pull requests
-
-Check the [Kedro repo to find open pull requests](https://github.com/kedro-org/kedro/pulls) and contribute a review!
-
-## Contribute a fix or feature
-
-If you're interested in contributing fixes to code or documentation, first read our [guidelines for contributing developers](https://kedro.readthedocs.io/en/stable/14_contribution/02_developer_contributor_guidelines.html) for an explanation of how to get set up and the process you'll follow.
-
-Once you are ready to contribute, a good place to start is to take a look at the `good first issues` and `help wanted issues` on [GitHub](https://github.com/kedro-org/kedro/issues).
-
-## Contribute to the documentation
-
-You can help us improve the [Kedro documentation online](https://kedro.readthedocs.io/en/stable/). Send us feedback as a [GitHub issue](https://github.com/kedro-org/kedro/issues) or start a documentation discussion on [GitHub](https://github.com/kedro-org/kedro/discussions).
-
-You are also welcome to make a raise a PR with a bug fix or addition to the documentation. First read the guide [Contribute to the Kedro documentation](https://kedro.readthedocs.io/en/stable/14_contribution/04_documentation_contributor_guidelines.html).
+We invite community members to join the TSC and help define the future of the Kedro project. Read the [guidance on becoming a Kedro maintainer](TODO) to understand the process of joining the TSC.
```

Comment:
Is this TODO still outstanding or should that link to the new technical_steering_committee chapter?